src/control/cmd/daos/pool.go

```diff
@@ -295,11 +295,12 @@ func convertPoolInfo(pinfo *C.daos_pool_info_t) (*daos.PoolInfo, error) {
 	return poolInfo, nil
 }
 
-func queryPool(poolHdl C.daos_handle_t, queryMask daos.PoolQueryMask) (*daos.PoolInfo, error) {
+func queryPoolHelper(poolHdl C.daos_handle_t, queryMask daos.PoolQueryMask) (*daos.PoolInfo, error) {
```

Comment:
naming suggestion - I'd leave `queryPool` as-is, as this is the function that calls `daos_pool_query`. Maybe the other function could be called `queryPoolRankLists`, since its main goal is to collect the different rank lists from multiple queries.

Reply:
OK